app/models/element_aviti_sequencing_request.rb

```diff
@@ -23,7 +23,7 @@ class ElementAvitiSequencingRequest < SequencingRequest
     custom_attribute(:read_length, integer: true, validator: true, required: true, selection: true)
     custom_attribute(:requested_flowcell_type, required: true, validator: true, selection: true)
 
-    custom_attribute(:percent_phix_requested, integer: true, required: true, minimum: 0)
+    custom_attribute(:percent_phix_requested, integer: true, required: true, maximum: 100)
```

Comment:
The percent_phix_requested maximum is set to 100 but the minimum was removed. The requirement is to have it in the range [0, 100].

Reply:
I see, it uses Attribute default.
```
def minimum
      @options.fetch(:minimum, 0)
    end
```